modules/ROOT/pages/major-changes.adoc

```diff
@@ -9,6 +9,38 @@ This list is in reverse chronological order to keep recent changes at the top.
 
 // To add a new change here, see the template at the end of the file.
 
+== Podman v5.0
+
+The Podman container runtime will be upgraded from v4 to v5. This is a https://blog.podman.io/2024/03/podman-5-0-has-been-released/[major release] that removes support for CNI networking in favor of Netavark.
+
+See also the https://fedoraproject.org/wiki/Changes/Podman5[Fedora Change] and the https://github.com/coreos/fedora-coreos-tracker/issues/1629[tracking issue].
+
+=== Planning
+
+This change will be rolled out together with the rebase to Fedora 40.
+
+|===
+|Update Stream |Targeted release date
+
+|`next` | 40.20240322.1.0
+|`testing`| To Be Determined
+|`stable`| Will follow `testing` as usual
+|===
+
+=== Notes
+
+The full release notes for Podman v5 are available https://github.com/containers/podman/releases/tag/v5.0.0[on GitHub]. Here is a summary of how this will impact Fedora CoreOS nodes:
+
+- CNI networking support has been removed and Netavark is now the only supported option. `podman system reset`
```

Comment:
looks like maybe some text got cut off here

we should go into more detail about where people can find instructions to migrate their system